### PR TITLE
fix(integ-tests): removed problematic logic in integ tests

### PR DESCRIPTION
**Pull request by FullstackGJJ**
Removed problematic parts of interactive workflow integ tests. Configured tests to run parallel to improve performance (From hours to 2 minutes).

The source of the issue came from the interactive endpoints when polled for the status, would sometimes give back a 202 and the results and have a header of `Cache-Control` equal to `max-stale=432000` so the test runner would go to sleep for 43200 seconds (12 hours) and the test runner would abort due to long sleeping time.

### CODE REVIEW

File: tests/src/test/java/Api/AfiInteractiveOptimizerEngineApiTests.java [hunk 1]
```diff
@@ -55,6 +57,8 @@ public void enginesApiGetOptimizationSuccess() throws ApiException, JsonProcessi
             AFIOptimizationParameters optimizationUnit = createUnitOptimization();
             AFIOptimizationParametersRoot afiOptimizerParam = new AFIOptimizationParametersRoot();
             afiOptimizerParam.setData(optimizationUnit);
+            Gson gson = new Gson();
+            System.out.println("afiOptimizerParam: " + gson.toJson(afiOptimizerParam));
```
Comment (saigiridhar21) [inline tests/src/test/java/Api/AfiInteractiveOptimizerEngineApiTests.java:61]: Shall we remove these two lines? Also, our library is dependant on `Jackson` and so, I feel that we shouldn't go ahead with `gson`. Can you also remove `gson` dependency in pom.xml?

**Review by saigiridhar21 — APPROVED** [review]
Left one minor comment.

File: tests/src/test/java/testconfig/PrintOutCurrentTestRunListener.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        System.out.println("testFinished "
+                + description.toString());
+    }
+
+    public void testRunFinished(Result result) {
+        System.out.println("testRunFinished " + result.toString()
+                + " time:"+result.getRunTime()
+                +" R"+result.getRunCount()
+                +" F"+result.getFailureCount()
+                +" I"+result.getIgnoreCount()
```
Comment (AkankshaAcharya) [inline tests/src/test/java/testconfig/PrintOutCurrentTestRunListener.java:29]: Can we add a colon similar to time in line 26 for better readability? And maybe a space after that?

File: tests/src/test/java/Api/AfiInteractiveOptimizerEngineApiTests.java (hunk 1 above)
Comment (christine-koul) [reply tests/src/test/java/Api/AfiInteractiveOptimizerEngineApiTests.java]: done https://github.com/factset/analyticsapi-engines-java-sdk/pull/63/commits/266e7b0894d01ab176698bb1d92877339de4456d

File: tests/src/test/java/testconfig/PrintOutCurrentTestRunListener.java (hunk 2 above)
Comment (christine-koul) [reply tests/src/test/java/testconfig/PrintOutCurrentTestRunListener.java]: done https://github.com/factset/analyticsapi-engines-java-sdk/pull/63/commits/453ecaead622869608e9edf4587f442e97649dbd

File: tests/pom.xml [hunk 3]
```diff
@@ -107,6 +129,12 @@
             <artifactId>jersey-apache-connector</artifactId>
             <version>${jersey-version}</version>
         </dependency>
+        <!--  Gson: Java to Json conversion -->
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.8</version>
+        </dependency>
```
Comment (AkankshaAcharya) [inline tests/pom.xml:137]: Can you remove this as its no longer needed as per Sai's comment?

**Review by AkankshaAcharya — APPROVED** [review]
Left one minor comment, but apart from that looks good.

File: tests/pom.xml (hunk 3 above)
Comment (christine-koul) [reply tests/pom.xml]: done https://github.com/factset/analyticsapi-engines-java-sdk/pull/63/commits/c61a100671f11f92d947d852c8d9c8274f7286d4

**Pull request merged**